crates/sound/Cargo.toml

```diff
@@ -23,6 +24,11 @@ vhost-user-backend = "0.8"
 virtio-bindings = "0.2"
 vm-memory = "0.10"
 vmm-sys-util = "0.11"
+pipewire = { version = "0.6.0", optional = true }
+libspa = "0.6.0"
+pipewire-sys = "0.6.0"
+libspa-sys = "0.6.0"
+bindgen = "0.64.0"
```

Comment:
They are only required when we enable `pw-backend`, so we need to set `optional = true` for all of them (e.g. `libspa = { version = "0.6.0", optional = true}`, etc.

Then we need to list all of them in on line 15 `pw-backend = ["pipewire", "libspa", ...`.

This will allow us to reduce the dependencies when some features are not enable at build time.

Reply:
ok, make sense thanks :)